.github/workflows/CI-coverage.yml

```diff
@@ -1,65 +1,61 @@
-name: CI-coverage
+name: Coverage
 
-run-name: "Workflow performing CI step: coverage"
+run-name: "Workflow CI/CD Steps: coverage"
 
 on:
   workflow_call:
 
 env:
-  RUST_BACKTRACE: 1
+  IMAGE_NAME: zencash/sc-ci-base
+  IMAGE_TAG: noble_rust-stable_latest
+  DOCKER_COMPOSE_CMD: "docker compose -f ${GITHUB_WORKSPACE}/ci/docker-compose.yml run --rm cargo-container"
+  DOCKER_BUILD_DIR: /build
+  DOCKER_CARGO_HOME: /tmp/.cargo
+  CARGO_BINARIES_INSTALL: "cargo-llvm-cov"
 
 jobs:
   coverage:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        # os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     steps:
-      - name: Checkout branch
+      - name: Git checkout
         uses: actions/checkout@v4
-      - name: Clean CI
```

Comment:
I see `clean_ci.sh` script has been removed, that was useful for cleaning up some unused stuff inside GitHubActions runner; indeed now I am getting some errors like `No space left on device`.

Reply:
Yes sure, it's on the `NH-core-ci_testing` repo.
https://github.com/HorizenLabs/NH-core-ci_testing/actions/runs/9366241226/job/25783220335

Reply:
I will add cleanup.sh script back. I see now what is going on. :+1: